acai-ios/CollectionViewCells/CheckoutMenuItemCollectionViewCell.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CheckoutMenuItemCollectionViewCell.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/12/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class CheckoutMenuItemCollectionViewCell: UICollectionViewCell {
+    
+    var menuItemTitleLabel: UILabel!
+    var menuItemPriceLabel: UILabel!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        menuItemTitleLabel = UILabel()
+        menuItemTitleLabel.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
when you use SnapKit, you don't need `.translatesAutoresizingMaskIntoConstraints = false` so you should just delete it.